migration/idempotent/001-base.sql

```diff
@@ -2233,16 +2227,15 @@ BEGIN
         SET LOCAL jit = DEFAULT;
     END IF;
 
-    UPDATE _prom_catalog.ids_epoch
-        SET (current_epoch, last_update_time) = (next_epoch, now())
-        WHERE current_epoch < next_epoch;
+    UPDATE _prom_catalog.global_epoch e
```

Comment:
I realize this isn't the final protocol implementation, nonetheless I want to point out that this update happens at a different point and with different locking in the proven protocol.